pycbc/workflow/grb_utils.py

```diff
@@ -710,14 +710,12 @@ def setup_pygrb_minifollowups(workflow, followups_file,
                      tags=tags)
     node = exe.create_node()
 
+    node.add_input_opt('--trig-file', resolve_url_to_file(trigger_file))
```

Comment:
Ideally, the `resolve_url_to_file` call would not happen here, but instead be done as soon as the trigger_file is read in (presumably a few levels above this). The help states that `trigger_file` is a `pycbc.workflow.File`, which is what we would want, but not what seems to be happening if this is needed.